src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -431,6 +437,30 @@ private static BoundPattern BindDiscardPattern(DiscardPatternSyntax node, TypeSy
             }
         }
 
+        private bool HasBlockedINumberConversion(Conversion patternConversion, TypeSymbol inputType)
```

Comment:
>HasBlockedINumberConversion [](http://example.com/codeflow?start=21&length=27)

The name feels somewhat misleading. We are deciding if we should block conversion, no one **has blocked** it yet, at least I didn't see any place like that.